### ISSUE TITLE
Styling: Post Details Page

### ISSUE THREAD

**Issue by niklasrammelkamp** [niklasrammelkamp/capstone-project#38]
## Value Proposition

**As a** User
**I want to** have a better UI
**so that** it's easier to understand the app.
​

## Description

​
<img width="363" alt="Bildschirm­foto 2023-02-10 um 14 27 47" src="https://user-images.githubusercontent.com/118187307/218103549-76a44981-1c55-4a9a-8a29-4d38af947b9b.png">

## Acceptance Criteria

- [ ] display all informations but styled
- [ ] display settings only if there are settings
- [ ] display button for see more comments if there are more than 5
      ​

## Tasks

- [ ] add styling to all components
- [ ] create function to display only settings that are given
- [ ] create function for display more comments if there are more than 5 and then each 5 comments
      ​

## Complexity

M

**Comment by Long-HaHoang** [comment]
Good scale. 
The second criterion is written a bit confusing to understand.
Maybe rephrase it a bit.

> If settings are available display setting options

That's just a minor thing. You get a ✅ for it.
